converter/internal/promtailconvert/testdata/remote_write.yaml

```diff
@@ -0,0 +1,44 @@
+clients:
+  - url: http://localhost/loki/api/v1/push
+    headers:
+      fat: ball
+      bird: house
+    tenant_id: sparrow
+    batchwait: 10s
+    batchsize: 1024
+    basic_auth:
+      username: robin
+      password: Worms123
+      password_file: /home/robin/.password
+    oauth2:
+      client_id: client_id
+      client_secret: client_secret
+      client_secret_file: foo/bar
+      scopes:
+        - scope1
+        - scope2
+      token_url: https://example/oauth2/token
+      endpoint_params:
+        host: example
+        path: /oauth2/token
+    bearer_token: "MySecretsAreSafeHere"
+    bearer_token_file: /home/robin/.special_token
+    proxy_url: http://proxy.example.com
+    tls_config:
+      ca_file: /home/robin/.ca
+      cert_file: /home/robin/.cert
+      key_file: /home/robin/.key
+      server_name: example.local
+      insecure_skip_verify: true
+    backoff_config:
+      min_period: 100ms
+      max_period: 10s
+      max_retries: 20
+    external_labels:
+      cluster: backyard-pine-treehouse-1
+      host: strawberry-pi
+    timeout: 30s
+tracing:
+  enabled: false
+server:
+  disable: true
```

Comment:
I've spent more time than I probably should on figuring out that this is actually an issue with the current promtail version that we have, which has been [fixed in this commit](https://github.com/grafana/loki/commit/de937e72cdb62062424e482fce1c87efc38bd9d9).

Once we update loki, this will start failing. It so happens that I am also the author of the PR updating loki, so I'll take care of broken tests at the point of whichever PR goes second.